htdocs/core/lib/fourn.lib.php

```diff
@@ -178,6 +178,11 @@ function supplierorder_admin_prepare_head()
 	$head[$h][1] = $langs->trans("SuppliersInvoice");
 	$head[$h][2] = 'invoice';
 	$h++;
+	
+	$head[$h][0] = DOL_URL_ROOT."/admin/supplier_payment.php";
+	$head[$h][1] = $langs->trans("SuppliersPayment");
+	$head[$h][2] = 'supplierpayment';
+	$h++;
```

Comment:
Payments are related to invoice. Why not adding option of payment into existing tab  /admin/supplier_invoice.php ?